docs/dev_guide/API-Mapping-query-guide.rst

```diff
@@ -0,0 +1,34 @@
+API-Mapping Query Guide
+=======================
+
+|tool_name| provides the functionality for users to query functionally
+compatible SYCL API for some CUDA API. The API query functionality can assist
+users on manual code migration and help users understand the SYCL API.

```

Comment:
Consider changing to "manual code migration, as well as help users understand the SYCL API" for flow

Reply:
fixed